From 0b4f21190173b5aa4b055a66cb7e505d70c70365 Mon Sep 17 00:00:00 2001
From: Christian Tismer <tismer@stackless.com>
Date: Fri, 7 Jun 2024 14:57:14 +0200
Subject: [PATCH 18/21] DEBIAN: Do the transition to Python 3.13

Task-number: PYSIDE-2751
Change-Id: I37b447148787e2923a58c091a5c8ac808d579bc0
Reviewed-by: Friedemann Kleint <Friedemann.Kleint@qt.io>
(cherry picked from commit ad18260e583a30505e42b04fd242c52ff36f735c)
---
 build_scripts/config.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/build_scripts/config.py b/build_scripts/config.py
index b2d6fc5..f02ae41 100644
--- a/build_scripts/config.py
+++ b/build_scripts/config.py
@@ -96,6 +96,7 @@ class Config(object):
             'Programming Language :: Python :: 3.10',
             'Programming Language :: Python :: 3.11',
             'Programming Language :: Python :: 3.12',
+            'Programming Language :: Python :: 3.13',
         ]
 
         self.setup_script_dir = None
-- 
2.52.0

